.idea/libraries/R_User_Library.xml

```diff
@@ -0,0 +1,6 @@
+<component name="libraryTable">
```

Comment:
you might want to ignore .idea directory

Reply:
Will do.

Reply:
After the dir is added to .gitignore the files have to be deleted as a followup commit.

Reply:
Ha. Working on it. I always forget how to get the files out of Git once it is added.